docs/lib/graphqlapi/fragments/android/authz/30_multi.md

```diff
@@ -13,7 +13,19 @@ Amplify.API.mutate(
 ```
 
 </amplify-block>
-<amplify-block name="Kotlin">
+<amplify-block name="Kotlin - Callbacks">
+
+```kotlin
+Amplify.API.mutate(
+    "[FRIENDLY-NAME-API-WITH-API-KEY]",
+    request,
+    { Log.i("MyAmplifyApp", "Mutation successful") },
```

Comment:
`Log.i()` vs `Log.d()` has been used inconsistently throughout this doc.  Can we stick to one? (I prefer `Log.i()`)